qdrant-landing/content/articles/vector-search-filtering.md

```diff
@@ -0,0 +1,726 @@
+
+Most people use default settings and build vector search apps that aren't properly configured or even setup for precise retrieval. In this guide, we will show you how to **use filtering to get the most out of vector search** with some basic and advanced strategies that are easy to implement. 
+
+#### Remember to run all tutorial code in Qdrant's Dashboard 
+
+The easiest way to reach that "Hello World" moment is to [**try filtering in a live cluster**](/documentation/quickstart-cloud/). Our interactive tutorial will show you how to create a cluster, add data and try some filtering clauses. 
+
+![qdrant-filtering-tutorial](/articles_data/vector-search-filtering/qdrant-filtering-tutorial.png)
+
+## Qdrant's approach to filtering 
+
+Let's take a look at this **3-stage diagram**. In this case, we are trying to find the nearest neighbour to the query vector **(green)**. Your search journey starts at the bottom **(orange)**.
+
+By default, Qdrant connects all your data points within the [**vector index**](/documentation/concepts/indexing/). After you [**introduce filters**](/documentation/concepts/filtering/), some data points become disconnected. Vector search can't cross the grayed out area and it won't reach the nearest neighbor. 
+How can we bridge this gap? 
+
+![filterable-vector-index](/articles_data/vector-search-filtering/filterable-vector-index.png)
+
+[**Filterable vector index**](/documentation/concepts/indexing/): This technique builds additional links **(orange)** between leftover data points. The filtered points which stay behind are now traversible once again. Qdrant uses special category-based methods to connect these data points. 
+
+### Qdrant's approach vs traditional filtering methods
+
+![stepping-lens](/articles_data/vector-search-filtering/stepping-lens.png)
+
+The filterable vector index is Qdrant's solves pre and post-filtering problems by adding specialized links to the search graph. It aims to maintain the speed advantages of vector search while allowing for precise filtering, addressing the inefficiencies that can occur when applying filters after the vector search.
+
+#### Pre-filtering reduces your search options
+In pre-filtering, a search engine first narrows down the dataset based on chosen metadata values, and then searches within that filtered subset. This reduces unnecessary computation over a dataset that is potentially much larger.
+
+> If the filters are too restrictive, you might sever links between points too early in the process. When the semantic search process begins, it won’t be able to travel to those locations. 
+
+**Figure 1:** We start with five products with different prices. First, the $1000 price **filter** is applied, narrowing down the selection of laptops. Then, a vector search finds the relevant **results** within this filtered set. Unfortunately, one result is missed. 
+
+![pre-filtering-vector-search](/articles_data/vector-search-filtering/pre-filtering.png)
```

Comment:
I propose to draw boxes around this too:

![image](https://github.com/user-attachments/assets/e768d052-41cd-41ac-b2df-d17784f05fc7)

Without it I was a bit confused - I thought it was about those specific entries in the middle, while it is about the whole set.